### DISCUSSION TITLE
Software's author changed - Rename package?

### DISCUSSION THREAD

**Discussion by ottnorml** [microsoft/winget-pkgs#62488] [Q&A] [answered]
Hello there,

is there a best practice to rename a package when the software's author changes? Background see https://github.com/Amebis/eduVPN/issues/193.
> The eduVPN project originated at SURF, but it migrated to GÉANT later.

**Comment by russellbanks** [comment]
I've recently done this with another package. I went about it by removing all old packages and submitting the package under the new author.

Example of removing the old version - #62914

Example of submitting the same package under new author - #62913

**Comment by denelon** [comment, marked as answer]
@ottnorml, a recent suggestion from @Trenly points a way forward:
* https://github.com/microsoft/winget-pkgs/issues/66937#issuecomment-1190154419
